### DISCUSSION TITLE
Mentor / Yardim

### DISCUSSION THREAD

**Discussion by alihansencan** [acikkaynak/deprem-yardim-projesi#54] [Ideas]
Uzun vadede olabilecek bir proje olabilir. Depremden etkilenen cocuklar ve genclere mentorlük, derslerde yardim ve programlama vs. gibi konulari ögretip yardimci olabilecegimizi düsünüyorum. Yardim etmek isteyen cok fazla insan var ve bu tarz bir projeyle hepimizin pek cok insana ulasabilecegini ve insanlara moral olarak destek olabilecegimizi düsünüyorum. Aklima akut olarak bir yardim fikri gelmediginden ve deprem bölgesinde olmadigimdam bu projeye elimden geldigince destek verebilirim. Ilgisini ceken arkadaslar olursa iletisime gecelim. Tesekkürler.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz Legal ekibin kontrolü sonrasında reddedildi.
Notları: "Legal ret, Bakanlık bünyesinde oluşturulabilir bir proje"
